Week3/solution2/Untitled6 (1)-checkpoint.ipynb

```diff
@@ -0,0 +1,965 @@
+    "conn=sqlite.connect(\"ems.db\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "id": "2b2d9e0f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v=\"select * from employee where ename like'_r%'\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 82,
+   "id": "679c440c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor=conn.cursor()\n",
+    "cursor.execute(v)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "id": "9ff956a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r=cur.fetchall()\n",
+    "    for i in range(len(r)):\n",
+    "        print(\"emp id:\",(r[i][0]))\n",
+    "        print(\"emp name:\",(r[i][1]))\n",
+    "        print(\"emp dept:\",(r[i][2]))\n",
+    "        print(\"emp basic:\",(r[i][3]))\n",
+    "        print(\"emp branch:\",(r[i][4]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 84,
+   "id": "4ad8183a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn.commit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 85,
+   "id": "33146fa2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn.colse()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 86,
+   "id": "768cb96b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 87,
+   "id": "cbff7952",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn=sqlite.connect(\"ems.db\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 88,
+   "id": "04d47e45",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "z=\"select * from employee where dept in('Account','Inventory') group by dept\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 89,
+   "id": "ebe97a64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor=con.cursor()\n",
+    "cursor.execute(z)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 90,
+   "id": "1f7e27a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r=cur.fetchall()\n",
+    "    for i in range(len(r)):\n",
+    "            print(\"emp id:\",(r[i][0]))\n",
+    "            print(\"emp name:\",(r[i][1]))\n",
+    "            print(\"emp dept:\",(r[i][2]))\n",
+    "            print(\"emp basic:\",(r[i][3]))\n",
+    "            print(\"emp branch:\",(r[i][4]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 91,
+   "id": "f869eff5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn.commit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 92,
+   "id": "3a14f86d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 93,
+   "id": "619f637e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qlite3"
```

Comment:
Is it "sqlite3" or "qlite3"?

Reply:
No, SQLite and SQLite3 are not the same.